src/index.ts

```diff
@@ -25,17 +25,17 @@ function blitz(filePath, compiler) {
   });
 }
 
-function logger(...msg) {
+function logger(...msg: string[]) {
   console.clear();
   console.log(new Date(), ...msg);
 }
 
-function runSandbox(filePath, compiler) {
+function runSandbox(filePath: string, compiler: any) {
   const code = compile(filePath, compiler);
   return code !== null ? newSandbox(code) : null;
 }
 
-function compile(filePath, { compile }) {
+function compile(filePath: string, { compile }: any) {
```

Comment:
maybe we should just take the compiler as an argument here (compiler: Service) and call compiler.compile?